### DISCUSSION TITLE
May I join team

### DISCUSSION THREAD

**Discussion by hipexscape** [ROM-builders/temporary#19420] [General]
My git username - acex69
My telegram - @raizel_69

My dt
https://github.com/acex69/dt-aosp
https://github.com/acex69/mt6768-common

Successful build (have more )
https://cirrus-ci.com/build/6082062084472832

**Comment by hipexscape** [comment]
@Apon77 sir

**Comment by Apon77** [comment]
Need more time, please follow #learn instruction from telegram group
